projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -214,7 +214,7 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked, OnDestro
               this.addressService.setMandatoryError(true);
               this.caseEditDataService.addFormValidationError({ id: `${path}_${path}`, message: `An address is required` });
             } else {
-              this.caseEditDataService.addFormValidationError({ id, message: `%FIELDLABEL% is required`, label });
+              this.caseEditDataService.addFormValidationError({ id, message: `%FIELDLABEL% Save and continue`, label });
```

Comment:
This seems to be an error - I don't see why we would want this change

Reply:
This text was used for test purposes only to demonstrate the Welsh translation enhancement works. The phrase "is required" is not currently in the Welsh database. As stated above, the PR is not to be merged.